### PR TITLE
Gem me baby

### PR DESCRIPTION
**Pull request by calebthompson**

### CODE REVIEW

File: spec/jellyfish/provisioner_spec.rb [hunk 1]
```diff
@@ -1,11 +1,66 @@
...
+        'provisioned'
+      end
+
+      def retire
+        'retired'
+      end
+    end
+  end
+
+  def order_item
```
Comment (masonforest) [inline spec/jellyfish/provisioner_spec.rb]: Set up FactoryGirl? Or too much work?

File: lib/jellyfish/provisioner.rb [hunk 2]
```diff
@@ -1,7 +1,41 @@
...
+
+    def self.retire(order_item_id)
+      perform(order_item_id, :warning_retirement_error) { |order_item| new(order_item).retire }
+    end
+
+    def initialize(order_item)
+      @order_item = order_item
+    end
+
+    # private
```
Comment (masonforest) [inline lib/jellyfish/provisioner.rb]: Should this be commented?

File: spec/jellyfish/provisioner_spec.rb (hunk 1 above)
Comment (calebthompson) [inline spec/jellyfish/provisioner_spec.rb]: Nothing to build. There's no real class to initialize.

File: lib/jellyfish/provisioner.rb (hunk 2 above)
Comment (calebthompson) [inline lib/jellyfish/provisioner.rb]: rubocop complains that you have a private keyword that's meaningless because class methods can't be made private.

**Comment by masonforest** [comment]
:palm_tree:

**Pull request merged**